apps/cookbook/src/app/examples/checkbox-example/examples/confirm.ts

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+
+const config = {
+  selector: 'cookbook-checkbox-confirm-example',
+  template: `<kirby-checkbox attentionLevel="1" checked="true" labelText="Confirm terms"></kirby-checkbox>`,
```

Comment:
Consider binding to `checked` input (as this will not work if changing to `false` because of js truthy value on string, which could lead to confusion for target developers...)
```suggestion
  template: `<kirby-checkbox attentionLevel="1" [checked]="true" labelText="Confirm terms"></kirby-checkbox>`,
```